res/app/components/stf/keys/add-adb-key/add-adb-key.pug

```diff
@@ -36,4 +36,7 @@
         i.fa.fa-plus.fa-fw
         span(translate) Add Key
 
+      br
```

Comment:
Yeah, I forgot to remove these. Need to add some margin before error-message alert box.